dm/tests/openapi/client/openapi_task_check

```diff
@@ -176,6 +176,72 @@ def get_task_list(task_count):
     assert data["total"] == int(task_count)
 
 
+def pause_task_success(task_name, source_name):
+    url = API_ENDPOINT + "/" + task_name + "/pause"
+    resp = requests.post(
+        url=url,
+        json=[
+            source_name,
+        ],
+    )
+    assert resp.status_code == 200
+
+
+def resume_task_success(task_name, source_name):
+    url = API_ENDPOINT + "/" + task_name + "/resume"
+    resp = requests.post(
+        url=url,
+        json=[
+            source_name,
+        ],
+    )
+    assert resp.status_code == 200
+
+
+def operate_schema_and_table_success(task_name, source_name):
+    schema_url = API_ENDPOINT + "/" + task_name + "/sources/" + source_name + "/schemas"
```

Comment:
Just asking, do we have a test for `task_ name/source_name` with special character?

Reply:
yes, becasuse JSON allows for both escaped or non-escaped non-ascii characters.